database-postgresql/helpers.js

```diff
@@ -0,0 +1,62 @@
+
+const client = require('./connection');
+
+const getPhotos = (trailId, sortOrder, callback) => {
+  let sortOrderStatement = sortOrder ? `ORDER BY upload_date ${sortOrder}` : '';
+
+  let getPhotosPSQLStatement = `SELECT * FROM trailphotos WHERE trail_id = $1` + sortOrderStatement;
```

Comment:
Why is this let and not const? Style guide? Your overall decision? Just happened?

Reply:
fixed... thanks!